test/common/index.js

```diff
@@ -174,13 +173,10 @@ function childShouldThrowAndAbort() {
   });
 }
 
-function ddCommand(filename, kilobytes) {
-  if (isWindows) {
-    const p = path.resolve(fixturesDir, 'create-file.js');
-    return `"${process.argv[0]}" "${p}" "${filename}" ${kilobytes * 1024}`;
-  } else {
-    return `dd if=/dev/zero of="${filename}" bs=1024 count=${kilobytes}`;
-  }
+function ddCommand(filename) {
```

Comment:
Then why not call it `createZeroFilledFile`?

Reply:
Because I'm lazy?